bulgarian/taken/taken.md

```diff
@@ -0,0 +1,2 @@
+Обаждаш се на дъщеря си и й казваш - "Те идват. Ще бъдеш отвлечена. Ще имаш само няколко секунди. Огледай ги и изкрещи високо какви отличителни белези виждаш - брада, татуировка,  коса, дрехи. Имаш само един шанс."  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.